services/app-web/src/hooks/useString.ts

```diff
@@ -0,0 +1,15 @@
+import { useLDClient } from 'launchdarkly-react-client-sdk'
+import { featureFlags } from '../common-code/featureFlags'
+
+export const useString = () => {
+    const ldClient = useLDClient()
```

Comment:
Do we plan on using this string long-term? Or will it be removed once this flag is gone?